org.eclipse.richbeans.widgets/src/org/eclipse/richbeans/widgets/shuffle/ShuffleViewer.java

```diff
@@ -281,6 +284,7 @@ public void widgetSelected(SelectionEvent e) {
 		up.setEnabled(false);
 		up.setLayoutData(gd);	
 		up.addSelectionListener(new SelectionAdapter() {
+			@Override
         	public void widgetSelected(SelectionEvent e) {
```

Comment:
The formatting goes funny here in a way that makes this a bit confusing, could you sort that out while you're here?